### ISSUE TITLE
Use part division to indicate physical volumes

### ISSUE THREAD

**Issue by rbeezer** [rbeezer/mathbook#1305]
Use the `part` division as a way to indicate physical volumes desired from PDF output.  In other words, a `book` with three `part` could generate a PDF with three Tables of Contents, three indices, index locators with an indication of the right part, etc.  Maybe `xref` that go beyond current ability to include cross-part part numbers.  Some appencies might only belong to one `part`, others might belong to each `part`.

Use this issue to add considerations not mentioned.

Mentioning @sean-fitzpatrick , @Alex-Jordan, @twjudson

**Comment by kcrisman** [comment]
>
> Use the part division as a way to indicate physical volumes desired from
> PDF output.
>

As a non-default option, correct?

**Comment by sean-fitzpatrick** [comment]
Yes. So that a book with parts could easily be split. (E.g. for us, APEX Calculus covers Calculus 1, 2, 3, and 4.) For students buying POD coursepacks, a publisher might want to be able to easily split the print edition while retaining index and contents, and without breaking cross references.

**Comment by sean-fitzpatrick** [comment]
@rbeezer if you can figure out a way to (for example) have the same appendix appear in multiple parts (volumes) without descending into a mess of multiply-defined references, you'll have PreTeXt one step ahead of LaTeX.

This was always a big problem for me with APEX in LaTeX. There was a bit of material that got repeated in two volumes (officially it was meant to be taught in Calc 3 but nobody was willing to trust that the as Calc 3 instructor would get to it before time ran out, so I had to include it in Calc 4). Eventually I solved it with reference labels that lived in nested `ifthen` statements.

References would come up multiply defined because the content was in the current document and the externally referenced document. Of course, most of the problems arose from the fact that each volume had its own top level files with unwanted sections commented out, with the `xr` package pulling in references from the other volumes.
It sounds like the PreTeXt approach would be more like building the whole book and then expecting the parts we want?

**Comment by kcrisman** [comment]
>
> Yes.
>
 Just checking!

**Comment by mitchkeller** [comment]
This is definitely of interest for the Active Calculus activities workbooks, and I hope we will eventually unify single-variable and multivariable into a single repo and produce one master HTML version but be able to use this to split out the print.
